woocommerce/woocommerce-gutenberg-products-block
# Convert renderFrontend to TypeScript

Refactors `render-frontend.js` to TypeScript.

### Manual Testing

Any regressions should be found by automated tested, but just in case:

1. Add the Cart or Checkout blocks to a post or page. 
2. Go to the frontend and verify they render correctly.
